File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task3_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.util.HashSet;
import java.util.Set;

public class Pangram {
    public boolean check(String sentence) {
        Set<Character> alphabet = new HashSet<>();
        for (char c : "abcdefghijklmnopqrstuvwxyz".toCharArray()) {
            alphabet.add(c);
        }
        for (char c : sentence.toLowerCase().replaceAll("[^a-z]", "").toCharArray()) {
            alphabet.remove(c);
        }
        return alphabet.isEmpty();
    }

    public static void main(String[] args) {
        Pangram pangram = new Pangram();
        System.out.println(pangram.check("The quick brown fox jumps over the lazy dog")); // true
        System.out.println(pangram.check("Hello World")); // false
    }
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task3_PERPLEXITY_llama-3-sonar-large-32k-chat.c
#include <ctype.h>
#include <stdbool.h>
#include <stdio.h>
#include <string.h>

bool check(char *sentence) {
    bool alphabet[26] = {false};
    for (int i = 0; sentence[i] != '\0'; i++) {
        char c = tolower((unsigned char)sentence[i]);
        if (c >= 'a' && c <= 'z') {
            alphabet[c - 'a'] = true;
        }
    }
    for (int i = 0; i < 26; i++) {
        if (!alphabet[i]) {
            return false;
        }
    }
    return true;
}

int main() {
    printf("%d
", check("The quick brown fox jumps over the lazy dog")); // 1
    printf("%d
", check("Hello World")); // 0
    return 0;
}